src/components/idleMembers/section/index.tsx

```diff
@@ -1,17 +1,22 @@
 import { FC } from 'react';
 import Card from '@/components/idleMembers/card';
 import classNames from '@/components/idleMembers/section/section.module.scss';
+interface userNameImage  {
```

Comment:
I find this interface name a little confusing. Is this for the image of the user's name?

Reply:
No, it isn't for the image of the user's name. The interface name was a bit confusing. It's resolved in my latest Commit.